submissions/pozzitive11/memoty-pair-game/js/index.js

```diff
@@ -49,34 +49,38 @@ const gameContainer = document.querySelector(".game__container");
 gameContainer.innerHTML = gameCard.join("");
 const gameCards = document.querySelectorAll(".game__card");
 
-let counter = 0;
-let arr = [];
-
-function flipCard() {
-  if (this && this.classList.contains("game__card")) {
-    this.classList.add("flip");
-
-    arr.push(this);
-  }
-  checkArrLength();
-
-  winCOndition();
-}
+let numberOfMatches = 0;
+let ArrayOfFlippedCards = [];
+
+gameCards.forEach((card) =>
+  card.addEventListener("click", ({ target }) => {
+    const parentTarget = target.parentElement;
+    if (parentTarget && parentTarget.classList.contains("game__card")) {
+      parentTarget.classList.add("flip");
+
+      ArrayOfFlippedCards.push(parentTarget);
+    }
+    checkArrayOfFlippedCardsLength();
+    checkForWin();
+  })
+);
 
-function checkArrLength() {
-  if (arr.length % 2 === 0) {
+function checkArrayOfFlippedCardsLength() {
+  if (ArrayOfFlippedCards.length % 2 === 0) {
```

Comment:
Again, why not `=== 2`?

Reply:
there was an idea 😺 , but I reworked it and now it equates as you say